db/migrations/postgres/000082_threadmemberships_seenreplycount.up.sql

```diff
@@ -0,0 +1,17 @@
+ALTER TABLE threadmemberships ADD COLUMN IF NOT EXISTS seenreplycount bigint DEFAULT 0;
```

Comment:
In my local DB I found some ThreadMemberships with no corresponding Threads. I'm not sure how this could have happened, but I've added the `Default 0` to guard against subsequent query failures if seenreplycount remains `NULL`